examples/jwt-issue-validate/jwt_issue_validate.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on how to how to issue and validate a JWT.
```

Comment:
The description has to be 50-160 characters (this is below 50)

Reply:
Updated with https://github.com/ballerina-platform/ballerina-distribution/pull/551/commits/f962ab67f771beb9643e82826fbf4e5282cb05ec